File: typescript/src/model/container.ts
export class Container {
    constructor(
        public uid: string
    ) { }

    toMap(): Record<string, any> {
        return {
            'UID': this.uid,
        };
    }

    static fromMap(data: Record<string, any>): Container {
        const uid = data['UID'];
        if (uid == null) {
            throw new Error('UID is not defined in the map');
        }
        return new Container(uid);
    }
}

File: python/openral_py/model/container.py
from typing import Any, Dict


class Container:
    def __init__(self, uid: str):
        self.uid = uid

    def to_map(self):
        return {
            'UID': self.uid,
        }

    @staticmethod
    def from_map(data: Dict[str, Any]) -> 'Container':
        uid = data['UID'] #throws Keyerror if UID is not present
        return Container(uid=uid)